web_src/js/features/repo-diff.ts

```diff
@@ -19,15 +19,40 @@ function initRepoDiffReviewButton() {
   const counter = reviewBox.querySelector('.review-comments-counter');
   if (!counter) return;
 
-  $(document).on('click', 'button[name="pending_review"]', (e) => {
-    const $form = $(e.target).closest('form');
-    // Watch for the form's submit event.
-    $form.on('submit', () => {
-      const num = parseInt(counter.getAttribute('data-pending-comment-number')) + 1 || 1;
+  function handleFormSubmit(form, textarea) {
```

Comment:
This architecture is a bit weird:
Why remove and re-add the handler all the time?
Typically, our behavior would rather be "initialize once on page load, and then keep it".